veggieseasons/lib/screens/details.dart

```diff
@@ -116,8 +121,12 @@ class DetailsScreen extends StatelessWidget {
             mainAxisSize: MainAxisSize.max,
             children: [
               Wrap(
-                children:
-                    veggie.seasons.map<Widget>((s) => SeasonCircle(s)).toList(),
+                children: Season.values
+                    .map((s) => SeasonCircle(
```

Comment:
multilined => with indentations etc are hard to read. Use {}

Reply:
Done.